.mergify.yml

```diff
@@ -1,38 +1,40 @@
+pull_request_rules:
+  - name: Automatic merge queue
+    conditions:
+      - -draft
+      - base = master
+    actions:
+      queue:
```

Comment:
the previous condition was automatically trying to queue every PR, even PRs that did not intend to be merged.

this will somewhat narrow the list of PRs set up for "queueing" (see screenshot) while still being flexible.

<img width="520" alt="Screenshot 2025-06-20 at 3 42 59 PM" src="https://github.com/user-attachments/assets/ac16960e-2e1d-4132-b4cb-4cc4bbd35f6a" />